runtime/resolvers/metricsview_summary.go

```diff
@@ -0,0 +1,136 @@
+package resolvers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"github.com/rilldata/rill/runtime"
+	"github.com/rilldata/rill/runtime/metricsview"
+	"github.com/rilldata/rill/runtime/pkg/mapstructureutil"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func init() {
+	runtime.RegisterResolverInitializer("metrics_summary", newMetricsSummaryResolver)
+}
+
+type metricsSummaryResolver struct {
+	runtime    *runtime.Runtime
+	instanceID string
+	mvName     string
+	executor   *metricsview.Executor
+	args       *metricsSummaryArgs
+}
+
+type metricsSummaryArgs struct {
+	Priority      int    `mapstructure:"priority"`
+	TimeDimension string `mapstructure:"time_dimension"` // if empty, the default time dimension in mv is used
```

Comment:
As per comment above, I think it should not accept any time dimension. It should summarize them all to help inform the AI about which to use.